lib/reporters/tap.js

```diff
@@ -7,7 +7,7 @@ function getSourceFromStack(stack, index) {
 		.split('\n')
 		.slice(index, index + 1)
 		.join('')
-		.replace(/^\s+at /, '');
```

Comment:
why did "at" have to be removed? Because `stack` here is cleaned up by `stack-utils`?